package.json

```diff
@@ -16,7 +16,7 @@
     "test:integration": "tap test/integration/*.js",
     "test:coverage": "tap test/{unit,integration}/*.js --coverage --coverage-report=lcov",
     "test:benchmark": "node test/benchmark/performance.js",
-    "test": "npm run test:lint && npm run test:unit && npm run test:integration",
+    "test": "npm run test:unit && npm run test:integration",
```

Comment:
Why separate `test:lint` from `test`? Seems like undoing this change and removing the separate `npm run test:lint` step in the workflow would work...?

Reply:
Heh - fair enough. I definitely agree that including linting in `npm test` is annoying when you're working on the tests!